admin.go

```diff
@@ -512,7 +512,7 @@ func (ca *clusterAdmin) ListPartitionReassignments(topic string, partitions []in
 
 	request.AddBlock(topic, partitions)
 
-	b, err := ca.findAnyBroker()
+	b, err := ca.Controller()
```

Comment:
Now this is using the controller you probably want to wrapper the body in a `return ca.retryOnError(isErrNoController, ...)` call like you did for `AlterPartitionReassignments` so it refreshes the cached controller if it is stale

Reply:
the `retryOnError` function cannot be used when a value needs to be returned. other functions in `admin.go` where a controller is used and a return type is needed also don't use retrying. How should this be handled? should we implement a retry function that returns something like `interface{}`?

Reply:
@dwi-di ah of course — let's just leave this for now and we can always follow-up in another PR